go.mod

```diff
@@ -17,10 +17,8 @@ require (
 	github.com/cosmos/cosmos-sdk v0.47.5
 	github.com/cosmos/gaia/v11 v11.0.0-00010101000000-000000000000
 	github.com/cosmos/gogoproto v1.4.10
-	// TEMP: Using this version to support duality swap-and-forward
-	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.0.0-20230629164013-34f5e666f806
-	// github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.0.0-20230803181732-7c8f814d3b79
-	github.com/cosmos/ibc-go/v7 v7.2.0
+	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.0.2-0.20231103132047-e5a274cf6fc2
```

Comment:
Is there a requirement to use this specific commit?
Can we just use the latest release?

Reply:
It does requires this https://github.com/cosmos/ibc-apps/pull/132 which looks like it never got added to a v7 release